docs/transports.md

```diff
@@ -1089,6 +1090,33 @@ const logger = pino({
 logger.error('<b>test log!</b>');
 ```
 
+<a id="pino-yc-transport"></a>
+### pino-yc-transport
+
+[pino-yc-transport](https://github.com/Jhon-Mosk/pino-yc-transport) is a Pino v7+ transport for writing to [Yandex Cloud Logging](https://yandex.cloud/ru/services/logging) from serveless functions or containers.
+
+```js
+const pino = require("pino");
+
+const config = {
+  level: "debug",
+  transport: {
+    target: "pino-yc-transport",
+  },
+};
+
+const logger = pino(config);
+
+logger.debug("some message")
+logger.debug({ foo: "bar" });
+logger.debug("some message %o, %s", { foo: "bar" }, "baz");
+logger.info("info");
+logger.warn("warn");
+logger.error("error");
+logger.error(new Error("error"));
+logger.fatal("fatal");
+```
+
 The `extra` parameter is optional. Parameters that the method [`sendMessage`](https://core.telegram.org/bots/api#sendmessage) supports can be passed to it.
```

Comment:
Does this line not go with the Telegram transport section?

Reply:
That's right, I fixed it. Thank you.